src/features/steps/simCardActivation/SIMCard.tsx

```diff
@@ -4,198 +4,54 @@
-    const [startRead, setStartRead] = useState(false);
-    const [failedRead, setFailedRead] = useState(false);
-    const [failedReset, setFailedReset] = useState(false);
-    const [iccid, setIccid] = useState('');
-
-    const formatIccid = (response: string) => {
-        const filteredResponse = response
-            .split('\n')
-            .filter(line => !!line && line.trim() !== 'OK')
-            .join('')
-            .trim();
-
-        const [, match] = filteredResponse.match(/%XICCID: (.{18}).*/) ?? [];
-
-        return match;
-    };
-
-    const reset = () =>
-        NrfutilDeviceLib.reset(device, 'Application', 'RESET_SYSTEM');
-
-    const delayedIccidRead = (
-        sp: Awaited<ReturnType<typeof getUARTSerialPort>>
-    ) =>
-        new Promise<void>((resolve, reject) => {
-            setTimeout(async () => {
-                try {
-                    const res = await sp.sendCommand('AT%XICCID');
-                    setIccid(formatIccid(res));
-                    resolve();
-                } catch (e) {
-                    reject();
-                }
-            }, 1000);
-        });
-
-    const readICCID = async () => {
-        let sp;
-        try {
-            sp = await getUARTSerialPort(device);
-            await sp.sendCommand('AT+CFUN=41');
-            await delayedIccidRead(sp);
-        } catch (e) {
-            sp?.unregister();
-            setFailedRead(true);
-            return;
-        }
-
-        try {
-            sp.unregister();
-            reset().catch(reset);
-        } catch (e) {
-            setFailedReset(true);
-        }
-    };
-
-    return (
-        <Main>
-            <Main.Content heading="Set up iBasis SIM card">
-                <p>
-                    If you are using the included iBasis SIM card, you need to
-                    register and enable it through nRF Cloud.
-                </p>
-                <p>Complete the following steps:</p>
+export default () => (
+    <Main>
+        <Main.Content heading="Set up iBasis SIM card">
+            <p>
+                If you are using the included iBasis SIM card, you need to
+                register and enable it through nRF Cloud.
+            </p>
+            <br />
+            <p>
+                If you have activated your iBasis SIM card before or are using a
+                SIM card from a different provider, you can skip this
+                configuration step.
+            </p>
+            <br />
+            <br />
+            <b>Complete the following steps in nRF Cloud:</b>
+            <br />
+            <br />
+            <ol className="tw-list-outside tw-list-decimal tw-pl-4">
+                <li>
+                    Log in or register a nRF Cloud account at{' '}
+                    <Link
+                        label="nrfcloud.com"
+                        href="https://nrfcloud.com/"
+                        color="tw-text-primary"
```

Comment:
Make the whole sentence a link.
Rephrase to:
```suggestion
Log in to nRF Cloud account or register to create one.
```